pkg/config/types/v1alpha1/network_config.go

```diff
@@ -41,6 +42,11 @@ func (n *NetworkConfig) Devices() []machine.Device {
 	return n.NetworkInterfaces
 }
 
+// Resolvers implements the Configurator interface.
+func (n *NetworkConfig) Resolvers() []string {
+	return n.NameServers
```

Comment:
The mismatch in naming here is a little confusing. If we decide on resolvers or nameservers, I think I'd prefer to stick with it all the way through.

Reply:
The mismatch is because we can't have a field and a method be named the same

Reply:
I know its redundant, and I'm hoping to fix eventually, but in other cases we just add something before what we really wanna call it. I care less about this right now, as I intend on clean the whole interface implementation up.